src/Symfony/Bridge/Doctrine/CacheWarmer/ProxyCacheWarmer.php

```diff
@@ -57,7 +57,9 @@ public function warmUp($cacheDir)
                 if (false === @mkdir($proxyCacheDir, 0777, true)) {
                     throw new \RuntimeException(sprintf('Unable to create the Doctrine Proxy directory "%s".', $proxyCacheDir));
                 }
-            } elseif (!is_writable($proxyCacheDir)) {
+            }
+
+            if (!is_writable($proxyCacheDir)) {
```

Comment:
this changes the code flow. The previous `if` block does not always stops the execution flow